cypress/e2e/GUI/loading.cy.js

```diff
@@ -0,0 +1,13 @@
+describe('Exibição do elemento de Loading', () => {
+  it('Deve exibir o elemento de Loading antes da busca inicial dos clientes', () => {
+    cy.intercept('GET', `${Cypress.env('API_URL')}/customers**`, {
+      delay: 1000,
+      fixture: false,
+    }).as('getCustomers');
+
+    cy.visit('/');
```

Comment:
Melhor deixar uma linha em branco entre a pré-condição e o resultado esperado.

Isso ajuda na legibilidade do código.

```suggestion
    cy.visit('/');
    
```